### PR TITLE
Add contact form on the Contact page

### PR DESCRIPTION
**Pull request by kimchirice**
## Rationale

Add contact form to the Contact page

## Advice for Reviewers & Testing Notes

- as discussed with @engramar and @QjStout, we came to agree to park current form as simple placeholder, we will apply update later when routes ready from backend.
- once used submitted the form with "name", "email" and "message", a console message will show details

## Screenshots:

- Screen Shot 2020-07-26 at 13 11 33

## Task Name
- Contact Page - Email form  #33 

## Linting Checklist
- [x] No commented code
- [x] Code Formatted nicely (Prettier)
- [x] PR your own code before you assign reviewers

### CODE REVIEW

File: client/src/components/ContactForm.vue [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+      </button>
+    </form>
+  </div>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      name: null,
```
Comment (promie) [inline client/src/components/ContactForm.vue:69]: These should be empty strings?

```
data() {
  return {
       name: '',
       email: '',
       message: ''
  }
}
```

File: client/src/components/ContactForm.vue [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+      email: null,
+      message: null
+    };
+  },
+  methods: {
+    formSubmit() {
+      console.log(`${this.name} from ${this.email} senting ${this.message}`);
```
Comment (promie) [inline client/src/components/ContactForm.vue:76]: Nice! the beauty of vue two way binding

File: client/src/components/ContactForm.vue (hunk 1 above)
Comment (kimchirice) [reply client/src/components/ContactForm.vue:69]: in fact, I made change, from '' to null

**Comment by engramar** [comment]
Looks good @kimchirice. Thanks!

**Comment by engramar** [comment]
If we are to go live soon without the backend. I think we can consider activating the smartforms the link to Calendly as of current as an interim. Below are the links to the current code. SmartForm is from Bootstrap Studio. =) 

[https://github.com/thecodesydney/codesydneywebsite/blob/master/templates/contact.html](https://github.com/thecodesydney/codesydneywebsite/blob/master/templates/contact.html)

[https://github.com/thecodesydney/codesydneywebsite/tree/master/static/js](https://github.com/thecodesydney/codesydneywebsite/tree/master/static/js)

**Pull request merged**